### PR TITLE
Universal Service Monitoring Doc Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Adds a section for NodeJS TLS Monitoring on the Setup page. https://github.com/DataDog/datadog-agent/pull/24285/

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/universal_service_monitoring/setup.md
Comment (guyarb) [inline content/en/universal_service_monitoring/setup.md:815]:
```suggestion
* Requires Agent version 7.54 or greater.
```

**Review by guyarb — COMMENTED** [review]
Thanks for taking care of it!

Overall looks good to me (left comments), but it will take about 7-8 weeks before we can merge this change (we need to wait for 7.54 release)

**Review by maycmlee — APPROVED** [review]

**Comment by guyarb** [comment]
please DO NOT MERGE

**Comment by alai97** [comment]
@guyarb noticed the [7.54.0 release](https://github.com/DataDog/datadog-agent/releases) went out yesterday, are we good to merge this in? :)

**Pull request merged**
